### PR TITLE
version number from cmakelists

### PR DESCRIPTION
**Pull request by emilljungberg**
New tool to show version number

### CODE REVIEW

File: src/version.h.in [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Get version number from CMAKE
+#ifndef INCLUDE_GUARD
```
Comment (spinicist) [inline src/version.h.in:2]: Can you swap this to `#pragma once` please? That's what we've used elsewhere in riesling

File: src/version.h.in [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+#define INCLUDE_GUARD
+
+#define PROJECT_NAME "@PROJECT_NAME@"
+#define PROJECT_VER  "@PROJECT_VERSION@"
+#define PROJECT_VER_MAJOR "@PROJECT_VERSION_MAJOR@"
```
Comment (spinicist) [inline src/version.h.in:7]: If we're using the `VERSION "string"` CMake function, do we need the `_VER_MAJOR`/`MINOR`/`PATCH`? I vote we don't.

File: src/version.h.in (hunk 1 above)
Comment (emilljungberg) [reply src/version.h.in:2]: Yes of course!

File: src/version.h.in (hunk 2 above)
Comment (emilljungberg) [reply src/version.h.in]: Agree!

**Comment by emilljungberg** [comment]
Made the changes in e8009c2.

**Comment by spinicist** [comment]
Great! I've merged this.

**Pull request merged**